jobs/promptLicenceCreationService.ts

```diff
@@ -47,10 +47,10 @@ export default class PromptLicenceCreationService {
   ): Promise<EmailContact[]> => {
     const managedCases = [...urgentPromptCases, ...initialPromptCases]
 
-    const mapPrisonerToReleaseCase = (prisoner: CvlPrisoner) => {
+    const mapPrisonerToReleaseCase = (prisoner: CvlPrisoner & { crn: string }) => {
       return {
         name: convertToTitleCase(`${prisoner.firstName} ${prisoner.lastName}`),
-        crn: undefined as string, // FIXME - this is never worked!
+        crn: prisoner.crn,
```

Comment:
We enrich the prisoner data before this so crn is available - its just the types were incorrect and a bit bodged previously